ssh/ssh.go

```diff
@@ -388,3 +389,20 @@ func (ctx *SSHContext) SetPermissions(host Host, path string, permissions string
 
 	return nil
 }
+
+func (ctx *SSHContext) WaitForMountPoints(host Host, path string) (err error) {
+	cmd, err := ctx.SudoCmd(host, "/run/current-system/sw/bin/systemd-run", "--wait", "--property=RequiresMountsFor="+path, "true")
```

Comment:
To avoid leaking services:

```suggestion
	cmd, err := ctx.SudoCmd(host, "/run/current-system/sw/bin/systemd-run", "--collect", "--wait", "--property=RequiresMountsFor="+path, "true")
```